README.md

```diff
@@ -40,6 +40,7 @@ For more information, you can refer to [ChainerRL's documentation](http://chaine
 |:----------|:---------------:|:----------------:|:---------------:|:------------------:|
 | DQN (including DoubleDQN etc.) | ✓ | ✓ (NAF) | ✓ | x |
 | Categorical DQN | ✓ | x | ✓ | x |
+| IQN | ✓ | x | ✓ | x |
```

Comment:
Does it support continuous action space?

Reply:
No, it does not support continuous actions for now.

Reply:
Got it.